src/cli/analyze/analyze-cli-command.ts

```diff
@@ -37,6 +37,33 @@ Please follow and contribute to the discussion at:
 		);
 	}
 
+	if (config.format === "webtypes") {
+		// Allow object being passed as JSON from command line
+		if (typeof config.webtypesConfig === "string") {
```

Comment:
`webtypesConfig` is typed as an object isn't it? surprised typescript is happy with this. maybe i read the wrong type?

tbh if we could avoid mutating `config` and instead build up another object we use, that'd be better imo. but i feel like you did this as an easy way to pass the state around rather than modfying function signatures

Reply:
Yes it's typed as object, but can be a string if built from command line. Maybe I should type it as `string | WebTypesTransformerConfig` and create a new attribute `parsedWebtypesConfig?: WebTypesTransformerConfig;` in AnalyzerCliConfig for holding the result of transformation
That would fix wrong typing and avoid strange mutation of this property. That would still do a mutation, but returning a new config would need more change on core application.

Reply:
yeah at run-time it can be a string, but the types say it should be an object, so i was surprised typescript was ok with that.

i would update the type to include `string` and do what you said (store the parsed config somewhere separate).